[release-v1.15] Apply vendor patch for rec-test

matzew opened the pull request:
vendoring this https://github.com/knative-extensions/reconciler-test/pull/762

pierDipi reviewed (commented):
For this one rather than patching, we can upgrade deps to release-1.15 of reconciler-test, wdyt ?

matzew commented:
yeah, that's fine - I can do that

On Fri, Sep 27, 2024 at 1:31 PM Pierangelo Di Pilato <
***@***.***> wrote:

> ***@***.**** commented on this pull request.
>
> For this one rather than patching, we can upgrade deps to release-1.15 of
> reconciler-test, wdyt ?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift-knative/eventing/pull/885#pullrequestreview-2333493984>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AABGPTT4X43FMJQZL6IFNGDZYU6Y5AVCNFSM6AAAAABO6N2SAGVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDGMZTGQ4TGOJYGQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Matthias Wessendorf

blog: http://matthiaswessendorf.wordpress.com/
sessions: http://www.slideshare.net/mwessendorf
twitter: http://twitter.com/mwessendorf

matzew commented:
```
	modified:   vendor/knative.dev/pkg/controller/stats_reporter.go
	modified:   vendor/knative.dev/pkg/webhook/stats_reporter.go
	modified:   vendor/knative.dev/reconciler-test/pkg/eventshub/103-pod.yaml
	modified:   vendor/knative.dev/reconciler-test/pkg/eventshub/104-forwarder.yaml
	modified:   vendor/knative.dev/reconciler-test/pkg/eventshub/resources.go
	modified:   vendor/knative.dev/reconciler-test/pkg/feature/feature.go
	modified:   vendor/knative.dev/reconciler-test/pkg/manifest/options.go
	modified:   vendor/knative.dev/reconciler-test/pkg/resources/cronjob/cronjob.go
	modified:   vendor/knative.dev/reconciler-test/pkg/resources/deployment/deployment.go
	modified:   vendor/knative.dev/reconciler-test/pkg/resources/deployment/deployment.yaml
	modified:   vendor/knative.dev/reconciler-test/pkg/resources/job/job.go
	modified:   vendor/knative.dev/reconciler-test/pkg/resources/pod/pod.go
```
